### PR TITLE
Feature/add+remove items

### PR DESCRIPTION
**Pull request by FrazierLE**
Summary of the work done:

- This completes creating the global variable we will need for POST
- Drop down menu value and quantity input value added to object with addButton 

Contributed to code:

- Name Lauren with a little Eleanor spice 🔥

Needs to review:

- Name Jenn & Kiko

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -131,6 +131,8 @@ function displayPantryPage() {
     navMessage.innerText = 'Pantry'
     hide([removeRecipeButton, pantryButton, allRecipes, singleRecipe, favoritesView, favoriteRecipeButton, favoriteButton, ingredientSidebar, filterSidebar])
     show([pantryView])
+    displayIngredientDropDown()
```
Comment (knpausch) [inline src/scripts.js:134]: good placement

File: src/scripts.js [hunk 2]
```diff
@@ -343,14 +345,14 @@ function displayIngredientDropDown() {
...
     })
 }
 
 function addOrRemoveToPantry(user) {
     pantryTable.innerHTML = ''
     const amount = apiIngredients.reduce((acc, value) => {
-        user.pantry.forEach(current => {
+        user.pantry.pantryData.forEach(current => {
             if(value.id === current.ingredient) {
             var object = {['Ingredient']: value.name, ['Amount']: current.amount}
```
Comment (knpausch) [inline src/scripts.js:357]: amazing logic here. Maybe in the future we could rename object to inventoryItem or pantryItem. I know the instructors get lil mad when we name objects 'objects'

File: src/scripts.js [hunk 3]
```diff
@@ -361,10 +363,19 @@ function addOrRemoveToPantry(user) {
...
 }
 
+function addItemToPantry() {
+    const foundIt = ingredients.ingredients.reduce((acc, element) => {
+        if(element.name === selectIngredient.value) {
+                acc = element.id
+        }
+            return acc
+        }, 0)
+        postItNote = {userID: user.id, ingredientID: foundIt, ingredientModification: inputQuantity.value}
```
Comment (knpausch) [inline src/scripts.js:378]: I console logged this and it came out just like the format, however according to the "Endpoints (GET and POST)" doc, I think ingredientModification is supposed to be a number not a string. We could do `Number(inputQuantity.value)` to change it to a number. Not important now, just a note.

**Review by knpausch — APPROVED** [review]
great job Lauren! you killed it on addItemToPantry and addOrRemoveToPantry

File: src/scripts.js (hunk 2 above)
Comment (jmyacobn) [reply src/scripts.js:357]: I like Kiko's idea of renaming the variable to pantryItem. We could always make a note of this for refactor, and do a find, replace all!

File: src/scripts.js (hunk 3 above)
Comment (jmyacobn) [reply src/scripts.js:378]: Good catch, Kiko!

**Review by jmyacobn — APPROVED** [review]
This looks great Lauren!

File: src/scripts.js (hunk 2 above)
Comment (FrazierLE) [reply src/scripts.js:357]: Oh yeah, I'll take that of that

File: src/scripts.js (hunk 3 above)
Comment (FrazierLE) [reply src/scripts.js:378]: Good catch, just updated it

**Pull request merged**